src/server/core/launch_server.c

```diff
@@ -30,6 +31,7 @@ static void listen_to_inputs(struct sockaddr_in *addr, int server_fd,
             free(timeout);
             return;
         }
+        detect_win(data);
```

Comment:
if someone wins i guess the server stops, you can do this by breaking the while loop